### DISCUSSION TITLE
Fetching .MRXS data via QuPath fails

### DISCUSSION THREAD

**Discussion by nmduonggg** [Project-MONAI/MONAILabel#1733] [Q&A]
Hi guys, 

I successfully start the MONAI label server for my pathology tasks, using QuPath. However, when I use file ".mrxs" to place in datasets folder, QuPath cannot fetch the WSI image. It works fine with the sample WSI image with format of ".svs". It seems that MONAI fails to support ".mrxs" format, which is supported by OpenSlide ?? 

Does anyone have suggestion on this ? Thanks guys

![image](https://github.com/user-attachments/assets/968cf008-540f-4d41-8385-1dc2b74c1a8b)
![image](https://github.com/user-attachments/assets/d8bdeef9-282d-4561-86d4-3286affbd2e9)

**Comment by diazandr3s** [comment]
Hi @nmduonggg,

Thanks for sharing this.
Here are the supported file formats: https://github.com/Project-MONAI/MONAILabel/blob/4736d5fab7a33edd264a30641759e6aa9b0d8153/monailabel/config.py#L72-L81
Do you know if there is an open-sourced ".mrxs" reader? Otherwise, is there an easy way to convert images from ".mrxs" to ".svs" format?

Hope this helps,
